### PR TITLE
Describe a way to override keyboard shortcuts in Grids

### PR DESCRIPTION
**Pull request by AlexJameson**
https://trello.com/c/D3ahSuUw/1235-datagrid-treelist-update-keyboard-support

### CODE REVIEW

File: concepts/05 Widgets/DataGrid/75 Keyboard Support.md [hunk 1]
```diff
@@ -85,6 +85,8 @@ A user can interact with the widget using the following keys:
     </table>
 </div>
 
+You can override these shortcuts or create your own ones using the [onKeyDown](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/onKeyDown.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onKeyDown') event handler.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/DataGrid/75 Keyboard Support.md]:
```suggestion
You can override these shortcuts or create your own shortcuts using the [onKeyDown](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/onKeyDown.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onKeyDown') event handler.
```

File: concepts/05 Widgets/TreeList/70 Keyboard Support.md [hunk 2]
```diff
@@ -83,6 +83,8 @@ A user can interact with the widget using the following keys:
     </table>
 </div>
 
+You can override these shortcuts or create your own ones using the [onKeyDown](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/onKeyDown.md '/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/#onKeyDown') event handler.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/TreeList/70 Keyboard Support.md]:
```suggestion
You can override these shortcuts or create your own shortcuts using the [onKeyDown](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/onKeyDown.md '/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/#onKeyDown') event handler.
```

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Widgets/GridBase/1 Configuration/onKeyDown.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ The widget's instance.
 #include common-ref-elementparam with { element: "widget" }
 
 ##### field(e.event): event
-#include common-ref-eventparam
+The event that caused the function to execute. It is a [dxEvent](/api-reference/50%20Common/Object%20Structures/dxEvent '/Documentation/ApiReference/Common/Object_Structures/dxEvent/') or a <a href="http://api.jquery.com/category/events/event-object/" target="_blank">jQuery.Event</a> when you use jQuery. The underlying native event is <a href="https://developer.mozilla.org/en-US/docs/Web/API/Document/keydown_event" target="_blank">keydown</a>.
```
Comment (arminal) [inline api-reference/10 UI Widgets/GridBase/1 Configuration/onKeyDown.md]: This paragraph makes no sense. The sentences don't tie into each other. Please rewrite.

Comment (AlexJameson) [reply api-reference/10 UI Widgets/GridBase/1 Configuration/onKeyDown.md]: Well, it is a commonly used description for the function parameter. We use so often that we moved it to a separate file -  #include common-ref-eventparam. But here I need to extend this description and mention that our event uses the native Javascript event - [onkeydown](https://developer.mozilla.org/en-US/docs/Web/API/Document/keydown_event) and extends it by adding some functionality. That's why I called it "an underlying event"
image

Can we say something like 'This event is based on the keydown native event'?

Comment (arminal) [reply api-reference/10 UI Widgets/GridBase/1 Configuration/onKeyDown.md]: ok, Julian said I can leave it as is too.

**Pull request merged**